packages/react-components/src/atoms/Toggle.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import css from '@emotion/css';
+import { lineHeight, perRem } from '../pixels';
+import { steel, paper, tin, charcoal } from '../colors';
+const checkboxLabelStyles = css({
+  display: 'flex',
+  height: '54px',
+  borderRadius: '27px',
+  border: `1px solid ${steel.rgb}`,
+  overflow: 'hidden',
+  backgroundColor: steel.rgb,
+  color: tin.rgb,
+  g: {
+    stroke: tin.rgb,
+  },
+});
+
+const buttonStyle = css({
+  width: '50%',
+  borderRadius: '27px',
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+});
+
+const iconStyles = css({
+  display: 'inline-block',
+  width: `${lineHeight / perRem}em`,
+  height: `${lineHeight / perRem}em`,
+  paddingRight: `${6 / perRem}em`,
+});
+const activeStyles = css({
+  backgroundColor: paper.rgb,
+  color: charcoal.rgb,
+  g: {
+    stroke: charcoal.rgb,
+  },
+});
+const checkboxStyles = css({
+  display: 'none',
+  '&:not(:checked) ~ .left': activeStyles,
+  '&:checked ~ .right': activeStyles,
+});
+const buttonRight = css({ marginLeft: '-20px', padding: '0 10px' });
+const buttonLeft = css({ marginRight: '-20px', padding: '0 10px' });
+
+interface ToggleProps {
+  defaultChecked?: boolean;
```

Comment:
I think this interface is confusing. Is this just a `checked` prop, what does the `default` mean? And if it is `checked`, does that mean `left` or `right` is enabled? I would assume `right`, but representing the selection using `'left'` or `'right'` may be more intuitive.